finetune/lm_sequence_labeling.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+
+from finetune.lm_base import LanguageModelBase, SEQUENCE_LABELING
+from finetune.target_encoders import SequenceLabelingEncoder
+from finetune.utils import sequence_predict
+from finetune.encoding import text_standardize
+
+
+class LanguageModelSequence(LanguageModelBase):
+
+    def __init__(self, autosave_path, verbose=True, label_pad_target="<PAD>"):
+        super().__init__(autosave_path=autosave_path, verbose=verbose)
+        self.label_pad_target = label_pad_target
+
+    def _text_to_ids_with_labels(self, *Xs):
+        question_answer_pairs, labels = self.encoder.encode_input_sequence_labeling(*Xs, max_length=self.hparams.max_length)
+
+        tokens, mask = self._array_format(question_answer_pairs)
+        padded_labels = []
+        for sequence in labels:
+            padded_labels.append(sequence + (self.hparams.max_length - len(sequence)) * [self.label_pad_target])
+
+        return tokens, mask, padded_labels
```

Comment:
Re: previous, prefer not to return many different outputs as ordered tuples. Consider dict w/ string keys instead.